RC: Presenting the new Fixed limitation per plan in our Docs (#3137)
RedisLabs/redislabs-docs: content/rc/subscriptions/create-fixed-subscription.md

         /rc/administration/setup-and-editing/create-subscription/
         /rc/administration/setup/create-subscription/
---

> Fixed Size subscription plans support low throughput workflows.  Several plans are available, each designed for different memory sizes.

Reviewer: @cmilesb how about:

> Fixed plans support low throughput workflows.

Author: ```suggestion
Fixed plans support low throughput workflows.  Several plans are available, each designed for different memory sizes.
```

When creating your subscription, you'll need to choose the plan that suits your needs.

If you're new to Redis Cloud, the [quick start]({{<relref "/rc/rc-quickstart.md">}}) helps you create an account with a free subscription and an initial database.  You also learn how to connect to your database.

## Fixed plans

> Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown in the tables below (updated February 2024). 

Reviewer: @cmilesb can we add a link to the marketing pricing [page](https://redis.com/cloud/pricing/) on the word "pricing"?

Author: ```suggestion
Fixed plan [pricing](https://redis.com/cloud/pricing/) scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown in the tables below (updated February 2024). 
```

The 30 MB Fixed plan is free; it's designed for learning and building test projects. It gives you enough space to learn Redis concepts and develop application prototypes.

> Our paid plans start at 250 MB and offer extra features, like multi-zone replication and backups (both daily and instant). They are great for bigger projects or production environments that require increased storage, reliability, and other features to support your operational needs.

Reviewer: @cmilesb I'm wondering whether "multi-zone replication" is right here. I think we should use "high availability" here, since replication is how we do it and availability is what the customer gets.

Author: ```suggestion
Our paid plans start at 250 MB and offer extra features, like high availability and backups (both daily and instant). They are great for bigger projects or production environments that require increased storage, reliability, and other features to support your operational needs.
```

If you need additional resources, you can [upgrade your subscription]({{<relref "/rc/subscriptions/view-fixed-subscription#upgrade-subscription-plan">}}) at any time.

> There are some differences between plans for the different replication options. See the tables for [No replication and single-zone replication](#cache-standard) and [Multi-zone replication](#multi-az) for more details.

Reviewer: @cmilesb Also here, I think we should use high availability rather than replication, especially since in the new packaging UI we use the former.

If you're new to Redis Cloud, the [quick start]({{<relref "/rc/rc-quickstart.md">}}) helps you create an account with a free subscription and an initial database.  You also learn how to connect to your database.

## Fixed plan subscription tiers

Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown in the tables below (updated February 2024). 

> The 30 MB Fixed plan is free; it's designed for learning and building test projects. It gives you enough space to learn Redis concepts and develop application prototypes.

Reviewer 2: Test --> please disregard
